<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>816</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>模式</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="dialogStyle" stdset="0">
   <string>BGStyle</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>641</width>
     <height>798</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetTop" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>病人信息</string>
             </property>
             <property name="lblStyle" stdset="0">
              <string>lblNormal</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClose">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>42</number>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtAnamnesisNO">
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>送检日期</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="txtTestDoctor">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>床    号</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDateTimeEdit" name="dtSampleSendTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="dtStyle" stdset="0">
         <string>dtNormalStyle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>性    别</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>检验医生</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="cmbBedNo">
        <property name="maxLength">
         <number>48</number>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="cmbDepartmentName">
        <property name="maxLength">
         <number>48</number>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QTextEdit" name="txtDiagnosis">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>审核医生</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>诊    断</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="cmbSampleSender">
        <property name="maxLength">
         <number>48</number>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbSexID"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_20">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPatientName">
        <property name="maxLength">
         <number>48</number>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="cmbWardName">
        <property name="maxLength">
         <number>48</number>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>送检科室</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>病 历 号</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>病    区</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>送检医生</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QTextEdit" name="txtRemark">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="showFull" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtAge"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmbAgeUnitID"/>
        </item>
       </layout>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="txtVerifyDoctor">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>年    龄</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px;</string>
        </property>
        <property name="text">
         <string>姓    名</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>备    注</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-left:10px</string>
        </property>
        <property name="text">
         <string>样本类型</string>
        </property>
        <property name="lblStyle" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmbBloodType"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="styleSheet">
       <string notr="true">color:blue;
font-weight:bold;
font-size:14;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
        <property name="btnStyle" stdset="0">
         <string>normalBtn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="btnStyle" stdset="0">
         <string>normalBtn</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtAnamnesisNO</tabstop>
  <tabstop>txtPatientName</tabstop>
  <tabstop>cmbSexID</tabstop>
  <tabstop>txtAge</tabstop>
  <tabstop>cmbAgeUnitID</tabstop>
  <tabstop>dtSampleSendTime</tabstop>
  <tabstop>txtDiagnosis</tabstop>
  <tabstop>txtRemark</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
